frontend/integration-tests/protractor.conf.ts

```diff
@@ -148,7 +149,7 @@ export const config = {
   skipSourceMapSupport: true,
   jasmineNodeOpts: {
     print: () => null,
-    defaultTimeoutInterval: 60000,
+    defaultTimeoutInterval: JASMSPEC_TIMEOUT,
```

Comment:
@miyamotoh This is where you are breaking. True you didn't modify the file in question, this however is being referenced there. TypeScript is seeing this value as a string OR number. I would assume this is likely because of `process.env.JASMINE_TIMEOUT` is an env variable  which all come in as a string. Perhaps simply cast / type `JASMSPEC_TIMEOUT`?

Reply:
Ah, great, thanks so much, @andrewballantyne for digging that out for me. I wonder if this is as good, earlier in the same file (line 19);
```
export const JASMSPEC_TIMEOUT = (!_.isNil(process.env.JASMINE_TIMEOUT) && _.isNumber(process.env.JASMINE_TIMEOUT) ? Number(process.env.JASMINE_TIMEOUT): 12000)
```

Reply:
I went with above, and got all CI checks pass. PTAL.